src/Command/Histogram.php

```diff
@@ -0,0 +1,440 @@
+<?php
+
+/** @noinspection PhpUnusedPrivateFieldInspection */
+/** @noinspection PhpMultipleClassDeclarationsInspection */
+
+declare(strict_types=1);
+
+namespace ArielAllon\RetsCli\Command;
+
+use ArielAllon\RetsCli\Configuration;
+use ArielAllon\RetsCli\DateTime\DateTimeRange;
+use ArielAllon\RetsCli\DateTime\DateTimeRangeInterface;
+use ArielAllon\RetsCli\Output\ListingsCsv;
+use ArielAllon\RetsCli\Output\StrategyInterface;
+use ArielAllon\RetsCli\PHRETS;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use PHRETS\Session;
+
+class Histogram extends Command
+{
+    private const ARGUMENT_KEY = 'key'; // mls
+    private const ARGUMENT_INTERVAL = 'interval'; // year, month, etc.
+    private const ARGUMENT_RESOURCE_ALIAS = 'resource_alias'; // listings, rooms, etc.
+
+    private const OPTION_OUTPUT = 'output';
+    private const OPTION_NO_PROGRESS_BAR = 'no-progress-bar';
+    private const OPTION_ADDITIONAL_QUERY = 'additional_query'; // TODO: Support this
+    private const OPTION_TIMESTAMP_FIELD = 'timestamp_field'; // pulled from config if not specified
```

Comment:
from what i can tell (though I could definitely be missing something), this doesn't fall back to the config.
which is fine if so, i would just remove the comment

```suggestion
    private const OPTION_TIMESTAMP_FIELD = 'timestamp_field';
```